docs/guide/headless-cms.md

```diff
@@ -36,30 +44,29 @@ Requesting and using [piece](/guide/pieces.md) data should seem familiar to deve
 REST API routes are automatically constructed using the application's base URL, an API prefix (`/api/v1/`), the name of a content type module, and possibly a document ID. If using an API key for authentication, that would be added at the end in a query parameter as normal. For example, making a request for an *`article`* piece type from a website at `https://example.rocks` would use a URL like:
 
 ```
-https://example.rocks/api/v1/article?apikey=000apikey123
+https://example.rocks/api/v1/article
 ```
 
 When using Apostrophe as a headless CMS, we will most frequently be making `GET` requests from the Apostrophe app (see the [pieces API reference](/reference/api/pieces.md) for more on other request types). There are two main types of `GET` requests we should think about: a "get all" request and a "get one" request.
 
 ### Get all the pieces!
 
-To make a **"get all"** request, hit the URL as described above. Using the native Node.js `fetch` utility, that might look like:
+To make a **"get all"** request, hit the URL as described above. Using the Node.js `node-fetch` utility, such a request in a front end application might look like:
```

Comment:
In a "front-end application" you would not need node-fetch at all, right? You'd just use window.fetch (aka just use fetch).

Reply:
Feels like we still need more context here. I think it might even be worth presenting a tiny little index.html vanilla js app that does the whole thing.

Reply:
True about node-fetch. Moving too quickly.

My goal here is to give the gist of how this works. We can open tickets to add more context, but I don't have something at hand to add as a full example and I think we need to be okay with done on this one for now.